Gemfile

```diff
@@ -206,6 +206,9 @@ group :development, :test do
   gem 'timecop'
   gem 'webmock'
   gem 'yard'
+  path 'modules' do
+    gem 'mocked_authentication'
```

Comment:
I put this here so that it's only in the `:development`, and `:test` groups. Do you think it's still worth putting in the existing `modules` and preventing it from other stacks another way?

I know we can block the routes by Rails environment but will we also be able to block the mock frontend from prod as well otherwise?

Reply:
ahhh I missed that it was in a different group. iirc `RAILS_ENV` in dev is `production` so we will need it in the production group. Different environments settings are controlled by the `vsp_environment` setting.

I think the condition on the mount should be good enough. It wont load anything from the engine
```ruby
mount MockedAuthentication::Engine, at: '/mocked_authentication' if Rails.env.development? || Settings.vsp_environment == 'development'
```

Reply:
I made the change to put it with the other modules

Reply:
Hmmm, looks like Github ate up my other comment but basically I had just said that 'I thought dev stack was `dev` environment but I think you're right that everything non-localhost is 'prod'